tables/variables.md

```diff
@@ -1 +1,71 @@
+# tables/variables
 
+The `Variables` table displays information concerning the various different `variables` that are stored on the database.
+The following `JOIN` queries can be carried out:
+
+- `geography_type_id` on the [topics](topics.md) table using `topic_id`.
+
+## What are top-level variables?
+
+`variables` relate to topics in a many to one relationship. Where every [topic](topics.md) is sub-divided into many different variables. These `variables` alongside their parent `topics` represent context of the database as without them the data would just be arbitrary numbers alongside geographical locations.
+
+## Example use
+
+I want to identify every child `variable` for the `topic` `SEX`. We will start by identidying the `id` for the chosen `topic` of `SEX`.
+
+```sql
+SELECT id, abbreviation, description  FROM c2011_meta.topics 
+WHERE abbreviation = SEX;
```

Comment:
This query doesn't run (missing quotes) - definitely worth running these quickly just as a sanity check.

Reply:
fair enough, idk how that ended up like that but good spot